integration-tests/gradle/src/test/java/io/quarkus/gradle/QuarkusGradleWrapperTestBase.java

```diff
@@ -23,10 +25,32 @@ public class QuarkusGradleWrapperTestBase extends QuarkusGradleTestBase {
 
     private Map<String, String> systemProps;
 
+    private static final List<Process> gradleProcesses = new ArrayList<>();
+    static {
+        Runtime.getRuntime().addShutdownHook(new Thread(QuarkusGradleWrapperTestBase::killProcesses));
+    }
+
     protected void setupTestCommand() {
 
     }
 
+    @AfterAll
+    static void killProcesses() {
+        gradleProcesses.forEach(Process::destroy);
```

Comment:
Ran into situations locally, when forked processes were still alive, so surefire forcefully terminated the test JVM. This after-all prevents the situation. (It doesn't kill a launched Gradle daemon, just the Gradle wrapper.)